### ISSUE TITLE
Contribute to the Knowledge base - Window 3

### ISSUE THREAD

**Issue by pcrossnz** [fergusfrl/RiverWiki#27]
I'm not sure if the mapping window is quite right for most folks to use without screwing it up. I certainly ballsed it up on my first attempt. It would be great if we could first: 
Name a marker,
Place a pin, 
Add details on feature option - This could be displayed in a pop up box, and a table somewhere. For example, text and image. 

We should talk about how best to do this. It's an important feature for users adding to the wiki

**Comment by fergusfrl** [comment]
Yup I agree. Will probably need to sit down and have a think about how we want to implement this feature
